rllib/utils/tests/test_framework_independent_components.py

```diff
@@ -0,0 +1,104 @@
+from abc import ABCMeta, abstractmethod
```

Comment:
Can you add this test to bazel? (actually are there other tests we aren't running anywhere?)

Reply:
Sure ...

Reply:
Yes: agents/ppo/test/test.py was never run.
I moved it to the new BUILD.bazel file in /rllib/